.fig/autocomplete/src/sync-ld-flags.ts

```diff
@@ -0,0 +1,283 @@
+      });
+    },
+  },
+};
+
+const projectOpt: Fig.Option = {
+  name: ["-p", "--project-key"],
+  description: "Project key",
+  isRepeatable: false,
+  isRequired: true,
+  icon: ICON_PROJECT,
+  priority: 800,
+  args: {
+    name: "string",
+    description: "Project key",
+    debounce: true,
+    generators: apiGenerators.projects,
+  },
+};
+
+const tokenOpt: Fig.Option = {
+  name: ["-t", "--api-token"],
+  description: "LaunchDarkly personal access token with write-level access.",
+  isRepeatable: false,
+  isRequired: true,
+  icon: ICON_API_TOKEN,
+  priority: 1000,
+  args: {
+    name: "string",
+    description: "API access token",
+    generators: {
+      script: `cat ~/.config/ldc.json`,
+      postProcess: getSuggestionsFromConfig('apitoken', ICON_API_TOKEN),
+    }
+  },
+};
+
+const hostOpt: Fig.Option = {
+  name: ["-H", "--host"],
+  description: "Hostname override",
+  isRepeatable: false,
+  icon: ICON_URI,
+  args: {
+    name: "URI",
+    description: "LaunchDarkly URI",
+    generators: {
+      script: `cat ~/.config/ldc.json`,
+      postProcess: getSuggestionsFromConfig('server', ICON_URI),
+    }
+  },
+};
+
+const sourceOpt: Fig.Option = {
+  name: ["-s", "--source-env"],
+  description: "Source environment",
+  dependsOn: ["-p"],
+  isRepeatable: false,
+  isRequired: true,
+  icon: ICON_ENV,
+  priority: 700,
+  args: {
+    name: "string",
+    description: "Environment key",
+    debounce: true,
+    generators: apiGenerators.environments,
+  },
+};
+
+const destinationOpt: Fig.Option = {
+  name: ["-d", "--destination-env"],
+  description: "Destination environment",
+  dependsOn: ["-p"],
+  isRepeatable: false,
+  isRequired: true,
+  icon: ICON_ENV,
+  priority: 600,
+  args: {
+    name: "string",
+    description: "Environment key",
+    debounce: true,
+    generators: apiGenerators.environments,
+  },
+};
+
+const completionSpec: Fig.Spec = {
+  name: "sync-ld-flags",
+  description: "Copy flag settings from one environment to another.",
+  options: [
+    {
+      name: ["-h", "--help"],
+      description: "Show help for sync-ld-flags",
+    },
+    projectOpt,
+    sourceOpt,
+    destinationOpt,
+    tokenOpt,
+    {
+      name: ["-o", "--omit-segments"],
+      description: "Omit segments when syncing",
+      isRepeatable: false,
+    },
+    {
+      name: ["-f", "--flag"],
+      description: "Sync only the specified flag",
+      isRepeatable: false,
+      icon: ICON_FLAG,
+      exclusiveOn: ["-T", "--tag"],
+      args: {
+        name: "string",
+        description: "Flag key",
+        debounce: true,
+        generators: apiGenerators.flags,
+      },
+    },
+    {
+      name: ["-T", "--tag"],
+      description: "Sync flags with the specified tag(s). Only flags with all tags will sync.",
+      icon: ICON_TAG,
+      exclusiveOn: ["-f", "--flag"],
+      args: {
+        name: "string",
+        description: "Tag name",
+      },
+    },
+    {
+      name: ["--dry-run"],
```

Comment:
Any reason why there's no shortcode for `dry-run`?

Reply:
i wasn't sure what a good shortcode would be since we already use `-d` and `-D` for other opts and i couldn't find any examples of what people typically use for this - open to suggestions!

Reply:
I looked around, both `make` and `yarn` uses `-n` as an alias for `--dry-run` so we can follow that.